functests/2_performance_update/updating_profile.go

```diff
@@ -30,6 +30,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 	chkKernel := []string{"uname", "-a"}
 	chkCmdLine := []string{"cat", "/proc/cmdline"}
 	chkKubeletConfig := []string{"cat", "/rootfs/etc/kubernetes/kubelet.conf"}
+	chkIrqbalance := []string{"grep", "-ls" + "ffffffff,fffffffe" + "/etc/sysconfig/irqbalance"}
```

Comment:
not sure the "+" is correct here. Shouldn't at least the last component (`/etc/sysconfig/irqbalance`) separated by a comma?

Reply:
Fixed. Thanks.